resources/content/schema/base/service.json

```diff
@@ -9,7 +9,10 @@
             "type": "int",
             "required": false,
             "nullable": true,
-            "default": 1
+            "default": 1,
+             "attributes" : {
+                "scheduleUpdate" : true
```

Comment:
For my own education, does these mean that any time this field is changed, an update process will be scheduled?